### PR TITLE
added api call for get plugins

### PR DESCRIPTION
**Pull request by t-chatoyan**

### CODE REVIEW

File: components/plugins/Breadcrumb.vue [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+        methods: {
+            breadcrumb() {
+                return [...new Set(this.slug.split("/")
+                    .filter(r => r !== ""))
+                ]
+            },
+            breadcrumbLink(item, index) {
+                return "/" + this.breadcrumb().slice(0, index + 1).join("/")
+            },
+            breadcrumbLinkExist(item, index) {
```
Comment (Skraye) [inline components/plugins/Breadcrumb.vue:31]: Missing some logic I think

File: components/plugins/Container.vue [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+        });
+        content = page
+    }
+
+    useContentHead(content);
+
+    const transformTitle = (text) => {
+        return text
+            .replace(/([A-Z])/g, '&#x200B;$1')
+            .replace(/([\u2700-\u27BF]|[\uE000-\uF8FF]|\uD83C[\uDC00-\uDFFF]|\uD83D[\uDC00-\uDFFF]|[\u2011-\u26FF]|\uD83E[\uDD10-\uDDFF])/g, '');
```
Comment (Skraye) [inline components/plugins/Container.vue:67]: Please add a description, not very readable for next people working on it

**Review by Skraye — CHANGES_REQUESTED** [review]
Strange number in the breadcrumb : 
image

Same exact call is being done multiple times (1 for each different type of task I think) when picking a plugin : 
image

File: components/content/ProseA.vue [hunk 3]
```diff
@@ -26,35 +26,38 @@
...
-            link = link.replace('../', '')
+        if (!route.path.includes("/plugins/")) {
+            const {data: page} = await useAsyncData(
+                `ProseA-${hash(route.path)}`,
+                () => queryContent(route.path).only("_file").findOne()
+            );
+            if (page.value) {
+                const routePath = route.path.replace(/\/$/, '')
+                const absolutePath = config.public.siteUrl + routePath;
+                if (link.match(/(\.\.\/){2,}/) && page.value._file.includes('index.md')) {
```
Comment (ridge-kimani) [inline components/content/ProseA.vue:37]: Let's add some comments to the conditions that resolve the regexes.

**Comment by brian-mulier-p** [comment]
Closing in favor of #762

**Pull request closed without merging**